Emby.Dlna/PlayTo/XMLUtilities.cs

```diff
@@ -0,0 +1,259 @@
+#pragma warning disable IDE0059 // Unnecessary assignment of a value : Left in to aid in debugging.
+
+using System;
+using System.Text;
+using System.Web;
+using XMLProperties = System.Collections.Generic.Dictionary<string, string>;
+
+namespace Emby.Dlna.PlayTo
```

Comment:
A XML parser that parses "invalid" XML code.

Whoever programmed Sharp TV's implementation of PlayTo couldn't be bothered to ensure the XML was valid, so HTML encoded it and wrapped it in a XML shell.